ui/src/time_picker.tsx

```diff
@@ -152,6 +156,11 @@ export const
       },
       // HACK: https://stackoverflow.com/questions/70106353/material-ui-date-time-picker-safari-browser-issue
       onOpen = () => setTimeout(() => (document.activeElement as HTMLElement)?.blur()),
+      onBlur = (ev: React.FocusEvent) => {
+        if (!ev.relatedTarget || !Fluent.elementContains(popperRef.current, ev.relatedTarget as HTMLElement)) {
```

Comment:
Why don't we want to always close the timepicker popper on blur?

Reply:
Because `onBlur` is also called when clicking on the popup buttons or when clicking on the dev console etc.

Reply:
> onBlur is also called when clicking on the popup buttons

Good to know. Thanks!